src/explore-education-statistics-common/src/components/form/FormTextSearchInput.tsx

```diff
@@ -24,6 +25,7 @@ const FormTextSearchInput = ({
 }: Props) => {
   const handleChange = debounce((event: ChangeEvent<HTMLInputElement>) => {
     if (onChange) {
+      logEvent(window.location.pathname, id, event.target.value);
```

Comment:
Really wouldn't put this in here as it's for all of the checkbox fields in table tool

Reply:
Why is that a problem?
Are you suggesting to remove it?

Reply:
I'll remove the logging from this component as there isn't any obvious value to logging the term, compared to the value in logging what options the user has selected in creating a table